File: C#/백준/Bronze/1076. 저항/저항.cs
using System;
using System.Collections.Generic;
namespace q{
    class w{
        static void Main(){
            List<string> a = new List<string>() {"black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white"};
            long sum=10*a.IndexOf(Console.ReadLine())+a.IndexOf(Console.ReadLine());
            long b = a.IndexOf(Console.ReadLine()); long c = 1;
            for(int i =0; i<b; i++){
                c*=10;
            }
            sum*=c;
            Console.WriteLine(sum);
        }
    }
}

File: Java/백준/Bronze/1076. 저항/저항.java
import java.util.Scanner;
import java.util.Arrays;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        String[] aa = {"black","brown","red","orange","yellow","green","blue","violet","grey","white"};
        int sum=10*Arrays.asList(aa).indexOf(in.next())+Arrays.asList(aa).indexOf(in.next());
        System.out.print(sum);
        int a = Arrays.asList(aa).indexOf(in.next());
        if(sum!=0){
            for (int i =0; i<a; i++){
                System.out.print('0');
            }
        }
    }
}